e2e/smartvol_ops_test.go

```diff
@@ -407,6 +407,54 @@ func testSmartVolumeDistributeDisperse(t *testing.T) {
 	checkZeroLvs(r)
 }
 
+func testSmartVolumeAutoDistributeReplicate(t *testing.T) {
```

Comment:
can you add some negative test cases?

Reply:
what kind of negative tests I can add? negative max-brick-size or max-brick-size is greater than request size?

Reply:
kind of invalid minimum and maximum values for  max-brick-size